xga/utils.py

```diff
@@ -647,11 +664,47 @@ def check_telescope_choices(telescope: Union[str, List[str]]) -> List[str]:
 else:
     eSASS_AVAIL = True
 
+# Then, we check to see what version of CIAO (if any) is installed - for the Chandra mission
+# Here we check to see whether CIAO is installed (along with all the necessary paths)
+CIAO_VERSION = None
+# This checks for an installation of Ciao
+CIAO_AVAIL = False
+
+ciao_out, ciao_err = Popen("ciaover -v", stdout=PIPE, stderr=PIPE, shell=True).communicate()
+# Just turn those pesky byte outputs into strings
+ciao_out = ciao_out.decode("UTF-8")
+ciao_err = ciao_err.decode("UTF-8")
+
+if "ciaover: command not found" in ciao_err:
+        warn("CIAO cannot be identified on your system, and Chandra data cannot be processed.")
```

Comment:
Just a aesthetics thing, but I prefer the way the warning output looks when you use the 'stacklevel=2' argument with it - I try to do that wherever I put warnings in XGA.

So this warning for instance would be:

```python 
warn("CIAO cannot be identified on your system, and Chandra data cannot be processed.", stacklevel=2)
```

Now that I look, I missed adding stacklevel=2 to some warnings in the utils, but I'll fix that.


Also, the warning message probably shouldn't be exactly the same as in DAXA - for instance the the eSASS one says "No eSASS installation detected on system, as such all functions in xga.generate.esass will not work." - so maybe something more like that